components/src/Tables/Table.js

```diff
@@ -1,38 +1,98 @@
 import React from 'react';
 import styled from 'styled-components';
+import { colour, space, font, radius, shadow } from '@nulogy/tokens';
```

Comment:
`shadow` is unused. Please remove.

In fact no tokens should be used in this file ... those values should come from the theme.

Reply:
Please replace with an import of the tokens package (`import tokens from '@nulogy/tokens`) + a default `theme` prop.

See [`Link.js`](https://github.com/nulogy/design-system/blob/master/components/src/Link/Link.js#L23-L25) for an example the default `theme` prop.